src/mongocxx/pipeline.hpp

```diff
@@ -80,6 +80,20 @@ class MONGOCXX_API pipeline {
     pipeline& limit(std::int32_t limit);
 
     ///
+    /// Performs a left outer join to an unsharded collection in the same database to filter in
+    /// documents from the “joined” collection for processing.
```

Comment:
I don't know if we care, but these quotes look non-ASCII to me. Does that matter?